[2025-04-07T19:56:41.479+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-07T19:56:41.501+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: meta_ads_elt_pipeline.dbt_processing.data_transformations.meta_ads_campaigns_run manual__2025-04-07T19:54:58.399691+00:00 [queued]>
[2025-04-07T19:56:41.508+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: meta_ads_elt_pipeline.dbt_processing.data_transformations.meta_ads_campaigns_run manual__2025-04-07T19:54:58.399691+00:00 [queued]>
[2025-04-07T19:56:41.509+0000] {taskinstance.py:2867} INFO - Starting attempt 2 of 3
[2025-04-07T19:56:41.521+0000] {taskinstance.py:2890} INFO - Executing <Task(DbtRunLocalOperator): dbt_processing.data_transformations.meta_ads_campaigns_run> on 2025-04-07 19:54:58.399691+00:00
[2025-04-07T19:56:41.535+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=4661) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-07T19:56:41.537+0000] {standard_task_runner.py:72} INFO - Started process 4666 to run task
[2025-04-07T19:56:41.538+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'meta_ads_elt_pipeline', 'dbt_processing.data_transformations.meta_ads_campaigns_run', 'manual__2025-04-07T19:54:58.399691+00:00', '--job-id', '240', '--raw', '--subdir', 'DAGS_FOLDER/meta_ads_dag.py', '--cfg-path', '/tmp/tmpb00xqgnu']
[2025-04-07T19:56:41.539+0000] {standard_task_runner.py:105} INFO - Job 240: Subtask dbt_processing.data_transformations.meta_ads_campaigns_run
[2025-04-07T19:56:41.580+0000] {task_command.py:467} INFO - Running <TaskInstance: meta_ads_elt_pipeline.dbt_processing.data_transformations.meta_ads_campaigns_run manual__2025-04-07T19:54:58.399691+00:00 [running]> on host ffa99f2ae5fe
[2025-04-07T19:56:41.649+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='admin' AIRFLOW_CTX_DAG_ID='meta_ads_elt_pipeline' AIRFLOW_CTX_TASK_ID='dbt_processing.data_transformations.meta_ads_campaigns_run' AIRFLOW_CTX_EXECUTION_DATE='2025-04-07T19:54:58.399691+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-07T19:54:58.399691+00:00'
[2025-04-07T19:56:41.651+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-04-07T19:56:41.651+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-04-07T19:56:41.651+0000] {logging_mixin.py:190} INFO - Current task name:dbt_processing.data_transformations.meta_ads_campaigns_run state:running start_date:2025-04-07 19:56:41.501386+00:00
[2025-04-07T19:56:41.652+0000] {logging_mixin.py:190} INFO - Dag name:meta_ads_elt_pipeline and current dag run status:running
[2025-04-07T19:56:41.652+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-07T19:56:42.661+0000] {local.py:213} INFO - dbtRunner is available. Using dbtRunner for invoking dbt.
[2025-04-07T19:56:42.662+0000] {local.py:416} INFO - Cloning project to writable temp directory /tmp/tmpdxmtvvtt from /opt/airflow/dags/dbt/dbt_biera
[2025-04-07T19:56:42.662+0000] {local.py:427} INFO - Partial parse is enabled and the latest partial parse file is /tmp/cosmos/meta_ads_elt_pipeline__dbt_processing__data_transformations/target/partial_parse.msgpack
[2025-04-07T19:56:42.668+0000] {config.py:360} INFO - Profile caching is enable.
[2025-04-07T19:56:42.678+0000] {base.py:84} INFO - Retrieving connection 'postgres_conn'
[2025-04-07T19:56:42.679+0000] {config.py:340} INFO - Profile found in cache using profile: /tmp/cosmos/profile/c99bf486c96f06e0233408d45300d7cde96c22a7e21e1f49350115e2a86b810b/profiles.yml.
[2025-04-07T19:56:42.679+0000] {runner.py:60} INFO - Trying to run dbtRunner with:
 ['run', '--models', 'meta_ads_campaigns', '--full-refresh', '--models', 'meta_ads_campaigns', 'meta_ads_ad_sets', 'meta_ads_ads', '--project-dir', '/tmp/tmpdxmtvvtt', '--profiles-dir', '/tmp/cosmos/profile/c99bf486c96f06e0233408d45300d7cde96c22a7e21e1f49350115e2a86b810b', '--profile', 'default', '--target', 'dev']
 in /tmp/tmpdxmtvvtt
[2025-04-07T19:56:42.747+0000] {logging_mixin.py:190} INFO - 19:56:42  Running with dbt=1.9.3
[2025-04-07T19:56:43.202+0000] {logging_mixin.py:190} INFO - 19:56:43  Registered adapter: postgres=1.9.0
[2025-04-07T19:56:43.850+0000] {logging_mixin.py:190} INFO - 19:56:43  [WARNING]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
There are 1 unused configuration paths:
- models.dbt_biera.analytics
[2025-04-07T19:56:43.968+0000] {logging_mixin.py:190} INFO - 19:56:43  Found 4 models, 4 sources, 434 macros
[2025-04-07T19:56:43.971+0000] {logging_mixin.py:190} INFO - 19:56:43
[2025-04-07T19:56:43.971+0000] {logging_mixin.py:190} INFO - 19:56:43  Concurrency: 1 threads (target='dev')
[2025-04-07T19:56:43.972+0000] {logging_mixin.py:190} INFO - 19:56:43
[2025-04-07T19:56:44.077+0000] {logging_mixin.py:190} INFO - 19:56:44  1 of 3 START sql view model staging.meta_ads_ad_sets ........................... [RUN]
[2025-04-07T19:56:44.189+0000] {logging_mixin.py:190} INFO - 19:56:44  1 of 3 OK created sql view model staging.meta_ads_ad_sets ...................... [CREATE VIEW in 0.11s]
[2025-04-07T19:56:44.191+0000] {logging_mixin.py:190} INFO - 19:56:44  2 of 3 START sql view model staging.meta_ads_ads ............................... [RUN]
[2025-04-07T19:56:44.231+0000] {logging_mixin.py:190} INFO - 19:56:44  2 of 3 OK created sql view model staging.meta_ads_ads .......................... [CREATE VIEW in 0.04s]
[2025-04-07T19:56:44.233+0000] {logging_mixin.py:190} INFO - 19:56:44  3 of 3 START sql view model staging.meta_ads_campaigns ......................... [RUN]
[2025-04-07T19:56:44.276+0000] {logging_mixin.py:190} INFO - 19:56:44  3 of 3 OK created sql view model staging.meta_ads_campaigns .................... [CREATE VIEW in 0.04s]
[2025-04-07T19:56:44.293+0000] {logging_mixin.py:190} INFO - 19:56:44
[2025-04-07T19:56:44.294+0000] {logging_mixin.py:190} INFO - 19:56:44  Finished running 3 view models in 0 hours 0 minutes and 0.32 seconds (0.32s).
[2025-04-07T19:56:44.345+0000] {logging_mixin.py:190} INFO - 19:56:44
[2025-04-07T19:56:44.346+0000] {logging_mixin.py:190} INFO - 19:56:44  Completed successfully
[2025-04-07T19:56:44.347+0000] {logging_mixin.py:190} INFO - 19:56:44
[2025-04-07T19:56:44.347+0000] {logging_mixin.py:190} INFO - 19:56:44  Done. PASS=3 WARN=0 ERROR=0 SKIP=0 TOTAL=3
[2025-04-07T19:56:44.894+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/cosmos/operators/local.py:616: UserWarning: The dataset URI postgres://postgres:5432/warehouse.source.meta_ads_ad_sets is not AIP-60 compliant: URI format postgres:// must contain database, schema, and table names. In Airflow 3, this will raise an exception.
  datasets = [Dataset(uri) for uri in uris]

[2025-04-07T19:56:44.894+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/cosmos/operators/local.py:616: UserWarning: The dataset URI postgres://postgres:5432/warehouse.source.meta_ads_ad_creatives is not AIP-60 compliant: URI format postgres:// must contain database, schema, and table names. In Airflow 3, this will raise an exception.
  datasets = [Dataset(uri) for uri in uris]

[2025-04-07T19:56:44.895+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/cosmos/operators/local.py:616: UserWarning: The dataset URI postgres://postgres:5432/warehouse.source.meta_ads_ads is not AIP-60 compliant: URI format postgres:// must contain database, schema, and table names. In Airflow 3, this will raise an exception.
  datasets = [Dataset(uri) for uri in uris]

[2025-04-07T19:56:44.895+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/cosmos/operators/local.py:616: UserWarning: The dataset URI postgres://postgres:5432/warehouse.source.meta_ads_campaigns is not AIP-60 compliant: URI format postgres:// must contain database, schema, and table names. In Airflow 3, this will raise an exception.
  datasets = [Dataset(uri) for uri in uris]

[2025-04-07T19:56:44.896+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/cosmos/operators/local.py:616: UserWarning: The dataset URI postgres://postgres:5432/warehouse.staging.meta_ads_ad_sets is not AIP-60 compliant: URI format postgres:// must contain database, schema, and table names. In Airflow 3, this will raise an exception.
  datasets = [Dataset(uri) for uri in uris]

[2025-04-07T19:56:44.896+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/cosmos/operators/local.py:616: UserWarning: The dataset URI postgres://postgres:5432/warehouse.staging.meta_ads_ads is not AIP-60 compliant: URI format postgres:// must contain database, schema, and table names. In Airflow 3, this will raise an exception.
  datasets = [Dataset(uri) for uri in uris]

[2025-04-07T19:56:44.897+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/cosmos/operators/local.py:616: UserWarning: The dataset URI postgres://postgres:5432/warehouse.staging.meta_ads_campaigns is not AIP-60 compliant: URI format postgres:// must contain database, schema, and table names. In Airflow 3, this will raise an exception.
  datasets = [Dataset(uri) for uri in uris]

[2025-04-07T19:56:44.897+0000] {local.py:465} INFO - Inlets: [Dataset(uri='postgres://postgres:5432/warehouse.source.meta_ads_ad_sets', extra=None), Dataset(uri='postgres://postgres:5432/warehouse.source.meta_ads_ad_creatives', extra=None), Dataset(uri='postgres://postgres:5432/warehouse.source.meta_ads_ads', extra=None), Dataset(uri='postgres://postgres:5432/warehouse.source.meta_ads_campaigns', extra=None)]
[2025-04-07T19:56:44.898+0000] {local.py:466} INFO - Outlets: [Dataset(uri='postgres://postgres:5432/warehouse.staging.meta_ads_ad_sets', extra=None), Dataset(uri='postgres://postgres:5432/warehouse.staging.meta_ads_ads', extra=None), Dataset(uri='postgres://postgres:5432/warehouse.staging.meta_ads_campaigns', extra=None)]
[2025-04-07T19:56:44.898+0000] {local.py:656} INFO - Assigning inlets/outlets with DatasetAlias
[2025-04-07T19:56:44.947+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-07T19:56:44.947+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=meta_ads_elt_pipeline, task_id=dbt_processing.data_transformations.meta_ads_campaigns_run, run_id=manual__2025-04-07T19:54:58.399691+00:00, execution_date=20250407T195458, start_date=20250407T195641, end_date=20250407T195644
[2025-04-07T19:56:44.956+0000] {taskinstance.py:3064} INFO - Creating event for DatasetModel(uri='postgres://postgres:5432/warehouse.staging.meta_ads_ad_sets', extra={}) through aliases "meta_ads_elt_pipeline__data_transformations__meta_ads_campaigns_run"
[2025-04-07T19:56:44.990+0000] {taskinstance.py:3064} INFO - Creating event for DatasetModel(uri='postgres://postgres:5432/warehouse.staging.meta_ads_ads', extra={}) through aliases "meta_ads_elt_pipeline__data_transformations__meta_ads_campaigns_run"
[2025-04-07T19:56:44.996+0000] {taskinstance.py:3064} INFO - Creating event for DatasetModel(uri='postgres://postgres:5432/warehouse.staging.meta_ads_campaigns', extra={}) through aliases "meta_ads_elt_pipeline__data_transformations__meta_ads_campaigns_run"
[2025-04-07T19:56:45.025+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-04-07T19:56:45.026+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-04-07T19:56:45.026+0000] {logging_mixin.py:190} INFO - Dag name:meta_ads_elt_pipeline queued_at:2025-04-07 19:54:58.405571+00:00
[2025-04-07T19:56:45.026+0000] {logging_mixin.py:190} INFO - Task hostname:ffa99f2ae5fe operator:DbtRunLocalOperator
[2025-04-07T19:56:45.078+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-07T19:56:45.108+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-07T19:56:45.109+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
